### PR TITLE
cert regression

### PR DESCRIPTION
**Pull request by jhou1**
Created new workflow for the regression testing, @rioliu-rh @jiajliu @shellyyang1989 please review.

### CODE REVIEW

**Comment by shellyyang1989** [comment]
Looks good to me:)

**Comment by jhou1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-4.10-upgrade-from-stable-4.9-aws-ipi-byo-iam-role-cert-rotation-p2-f14

File: ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-chain.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  steps:
+  - ref: cucushift-upgrade-helper-service-build-info
+  - ref: idp-htpasswd
+  - ref: cucushift-pre
+  - ref: openshift-extended-upgrade-pre
+  - ref: cucushift-upgrade-prepare
+  - ref: cucushift-upgrade-healthcheck
+  - ref: cucushift-upgrade-toimage
+  - ref: openshift-extended-upgrade-post
+  - ref: openshift-oc-certrotation
```
Comment (jiajliu) [inline ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-chain.yaml:12]: Refer to dev's [pr](https://github.com/openshift/release/pull/40388/files), certrotation ran before e2e test. Should we locate `openshift-oc-certrotation` before `openshift-extended-upgrade-post`? cc feature owner @sunzhaohua2 @xingxingxia @rioliu-rh to confirm.

Comment (xingxingxia) [reply ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-chain.yaml:12]: Agree.

**Comment by xingxingxia** [comment]
LGTM except above order issue.

File: ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-chain.yaml (hunk 1 above)
Comment (jhou1) [reply ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-chain.yaml:11]: Thanks, updated

**Comment by jiajliu** [comment]
lgtm

**Comment by jhou1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-4.10-upgrade-from-stable-4.9-aws-ipi-byo-iam-role-cert-rotation-p2-f14

**Comment by jhou1** [comment]
/hold
Waiting for a successful run.

**Comment by jhou1** [comment]
/close
in favor of #40731

**Pull request closed without merging**